iree/hal/cuda/graph_command_buffer.c

```diff
@@ -425,7 +425,17 @@ static iree_status_t iree_hal_cuda_graph_command_buffer_dispatch(
     uint32_t workgroup_x, uint32_t workgroup_y, uint32_t workgroup_z) {
   iree_hal_cuda_graph_command_buffer_t* command_buffer =
       iree_hal_cuda_graph_command_buffer_cast(base_command_buffer);
-
+  iree_hal_executable_layout_t* layout =
+      iree_hal_cuda_executable_get_layout(executable, entry_point);
+  iree_host_size_t num_constants =
+      iree_hal_cuda_executable_layout_num_constants(layout);
+  iree_host_size_t constant_base_index =
+      iree_hal_cuda_push_constant_index(layout);
+  // Patch the push constants in the kernel arguments.
+  for (iree_host_size_t i = 0; i < num_constants; i++) {
```

Comment:
do you validate the max num_constants is under the limit when creating the layout? (should match IREE_HAL_CUDA_MAX_PUSH_CONSTANT_COUNT, so you may want to move that to your executable_layout.h so you can share it in both)

Reply:
right, added that